web/styles/zulip.css

```diff
@@ -2407,13 +2407,13 @@ textarea.invitee_emails {
     }
 }
 
-#custom-expiration-time-input,
+.custom-input-time-value,
 #invite-user-form {
     margin: 0;
 }
 
-select.invite-expires-in,
-select.custom-expiration-time,
+select.input-time-choices,
+select.custom-input-time-unit,
```

Comment:
I am not sure whether we need to have this change. For other modals opened from settings we use `modal_select` class to apply this CSS.

We could also try changing the invite modal as well to use that class. I guess at the time we added `modal_select` class, invite UI used to be an overlay and not modal.

Reply:
I have removed this complete css and have added `modal_select` class to all select elements.